packages/api/src/resolvers/article/index.ts

```diff
@@ -289,6 +293,7 @@ export const createArticleResolver = authorized<
         readingProgressAnchorIndex: 0,
         state: ArticleSavingRequestStatus.Succeeded,
         language: parsedContent?.language,
+        wordsCount,
```

Comment:
So right now we save it but aren't exposing it in GQL, right? (That is fine with me)

Reply:
Yes